install/kubernetes/helm/istio/files/injection-template.yaml

```diff
@@ -46,6 +46,11 @@ initContainers:
     privileged: true
```

Comment:
This file should no longer be used - the new installer has a different tempalate in istio/installer, that's what we'll use.